auth.py

```diff
@@ -109,7 +109,7 @@ def get_client_creds(self, client, connection_name):
             'username_key': username_key,
             'client_id': client['client_id'],
             'client_secret': client['client_secret'],
-            'scopes': ['openid', 'name', 'profile', 'email']
+            'scope': ['openid', 'name', 'profile', 'email']
```

Comment:
Can you leave a note in the commit message about what this change is for?

Reply:
Sorry about this, it should have probably been better to open an issue or another PR about this.
But when I checked the logs to see if the regex works, I saw this error:
```
[W 2021-02-09 10:37:19.991 JupyterHub configurable:190] Config option `scopes` not recognized by `GenericOAuthenticator`.  Did you mean `scope`?
```
I checked with `oauthenticator` too:
https://github.com/jupyterhub/oauthenticator/blob/3d249bba92db34519c016e81c8281ace089ea01d/oauthenticator/oauth2.py#L266

Reply:
This is from the old z2jh `jupyterhub_config.py`

```python
set_config_if_not_none(c.OAuthenticator, "scope", "auth.scopes")
```

And this is from the deprecation logic which seem to be correct also, pweh!

```yaml
{{- define "jupyterhub.authDep.remapOldToNew.map" }}
scopes: OAuthenticator.scope
# ...
```